### ISSUE TITLE
restructure user installation section by operating system

### ISSUE THREAD

**Issue by janzandr** [EnMAP-Box/enmap-box-documentation#48]
_Discussed with @jakimowb._

It is proposed to structure the user installation section by operating system. This will introduce some redundancy, but will streamline the reading expirience of an individual user.

```
USER SECTION
Installation
    Windows
        OSGeo4W
            Install QGIS
            Install Python Dependencies
            Run QGIS and install EnMAP-Box Plugin
        Mambaforge
            Install QGIS and Python Dependencies
            Run QGIS and install EnMAP-Box Plugin
        Conda
            Install QGIS and Python Dependencies
            Run QGIS and install EnMAP-Box Plugin
    Linux
        ...
    MacOS
        ...

```

Subtasks:
- [x] Windows section
- [x] Linux section
- [ ] MacOS section

@jakimowb I would suggest, that I kick-off with the Windows section first, we discuss and reiterate until we like it, and afterwards you can take over and fill in the Linux and MacOS sections.

**Comment by janzandr** [comment]
@jakimowb you can take over and fill in the Linux and MacOS sections.
Here is the Windows section:
https://enmap-box.readthedocs.io/en/48-restructure-user-installation-section-by-operating-system/usr_section/usr_installation_new.html#qgis-installation-on-windows

**Comment by janzandr** [comment]
Just merged the Windows section. Reopen for MacOS and Linux.

**Comment by janzandr** [comment]
@jakimowb can you finalize the MacOS section before we release v3.14?

**Comment by jakimowb** [comment]
fixed by dbcdba30fab370eb39707db5d248aab6351a9f56
